app/views/alaveteli_pro/info_request_batches/_message_preview.html.erb

```diff
@@ -32,7 +32,8 @@
           <%=
             _('You can view the ' \
               '<a href="{{existing_batch_url}}">existing batch</a>, or ' \
-              '<a href="{{edit_batch_url}}">edit you request</a>.',
```

Comment:
Couple of minor tweaks:

* Drop the "or"  on L34
* Spelling typo: you -> you**r** request on L35
* tweak the final option to "or ignore this warning by confirming you want to proceed anyway."

So it should read:

> You can view the existing batch, edit your request, or ignore this warning by confirming you want to proceed anyway.